stream_alert_cli/manage_lambda/version.py

```diff
@@ -68,18 +68,17 @@ def _publish_helper(self, cluster=None):
 
         # Configure the Lambda client
         client = boto3.client('lambda', region_name=region)
-        code_sha_256 = self.config['lambda'][self.package.config_key]['source_current_hash']
 
         # Publish the function(s)
         # TODO: move the extra logic into the LambdaPackage subclasses instead of this
         if self.package.package_name == 'stream_alert_app':
-            if not 'stream_alert_apps' in self.config['clusters'][cluster]['modules']:
+            if 'stream_alert_apps' not in self.config['clusters'][cluster]['modules']:
```

Comment:
++ I assume your editor flagged this? 😂

Reply:
Yep. Both evaluate the same way, but this is the more proper form because it reads less ambiguous